### PR TITLE
i18n for Italian language

### LINKED ISSUE #38: Add i18n for Italian language
We need to add support for the Italian language translation.

## How

- Clone this repository
- Go to `public\locales` folder
- Open `it.json` file and translate each phrase from `en` to `it`
- Create a PR

Screenshot 2022-11-16 at 14 00 25

### PR DESCRIPTION
**Pull request by nicpuppa**
Closes #38

### CODE REVIEW

**Comment by Wabri** [comment]
Can you close the #32 ?

File: public/locales/en.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "when": "Coming soon",
     "length": "March 2023",
     "type": "Speakers",
-    "description": "We are still exploring your submissions so if you have an idea and you would like to talk about it head over to our CFP"
+    "description": "We are still exploring your submissions so if you have an idea and you would like to talk about it head over to our <link href='https://sessionize.com/opensourceday23/'>CFP</link>"
```
Comment (404answernotfound) [inline public/locales/en.json:89]: Attento qui, credo ci sia da mettere il rich text direttamente sul component in questo caso (per riferimento cerca file con link)

File: public/locales/en.json [hunk 2]
```diff
@@ -114,8 +114,8 @@
     "when": "Coming soon",
     "length": "March 2023",
     "type": "Venue",
-    "description": "Keep in touch through our channels (Discord, LinkedIn) to know which Venue will host us for this event :)",
-    "heading": "Want to know more about the Venue for this event? :) Keep yourself posted on LinkedIn, Twitter or better yet, <link>Discord</link>!"
+    "description": "Keep in touch through our channels (Discord, LinkedIn or Twitter) to know which Venue will host us for this event :)",
+    "heading": "Want to know more about the Venue for this event? :) Keep yourself posted on <link href='https://www.linkedin.com/company/schrodinger-hat/'>LinkedIn</link>, <link href='https://twitter.com/OSDayDev'>Twitter</link> or better yet, <link href='https://discord.gg/JrrrkUj4'>Discord</link>!"
```
Comment (404answernotfound) [inline public/locales/en.json:118]: Stessa cosa, c'è da mettere il `t.rich(traduzione, { funzione })`

File: public/locales/it.json [hunk 3]
```diff
@@ -35,90 +35,90 @@
...
+    "where": "Firenze",
     "when": "Coming soon",
-    "length": "March 2023",
+    "length": "Marzo 2023",
     "type": "Agenda",
-    "description": "We are still looking for Speakers and organising the event! Want to participate? :)",
-    "heading": "Head over to the CFP page or contact us directly @ <link>SH</link>",
-    "heading_2": "You can find all the community info on our website, <link>Schrodingers Hat</link> and if you have any questions about inclusivity, diversity or accessibility please ask us at any time!"
+    "description": "Stiamo ancora cercando Speakers e organizzando l'evento! Vuoi partecipare? :)",
+    "heading": "Visita la pagina della <link href='https://sessionize.com/opensourceday23/'>CFP</link> o contattaci direttamente @ <link href='mailto:schrodinger.hat.show@gmail.com'>SH</link>",
```
Comment (404answernotfound) [inline public/locales/it.json:52]: t.rich

File: public/locales/it.json [hunk 4]
```diff
@@ -35,90 +35,90 @@
...
-    "subtitle": "These are the great minds that will talk about tech and open source :) Coming soon",
-    "where": "Florence",
+    "subtitle": "Queste sono le grandi menti che parleranno di tecnologia e Open Source :) Coming soon",
+    "where": "Firenze",
     "when": "Coming soon",
-    "length": "March 2023",
+    "length": "Marzo 2023",
     "type": "Speakers",
-    "description": "We are still exploring your submissions so if you have an idea and you would like to talk about it head over to our CFP"
+    "description": "Stiamo ancora valutando le vostre proposte, quindi se avete un'idea e volete parlarne visitate la nostra <link href='https://sessionize.com/opensourceday23/'>CFP</link>"
```
Comment (404answernotfound) [inline public/locales/it.json:89]: t.rich

**Comment by 404answernotfound** [comment]
> Can you close the #32 with this pr ?

we miss a few rich-text formats because new links were added

**Comment by 404answernotfound** [comment]
Screenshot 2022-11-16 at 14 33 11

Check how Agenda.tsx component works

**Comment by nicpuppa** [comment]
> Can you close the #32 with this pr ?

I would wait to do this task until we have more information

**Pull request merged**